site/src/pages/ManagementSettingsPage/Sidebar.tsx

```diff
@@ -187,18 +178,20 @@ export const SidebarNavItem: FC<SidebarNavItemProps> = ({
 interface SidebarNavSubItemProps {
   children?: ReactNode;
   href: string;
+  end?: boolean;
 }
 
 export const SidebarNavSubItem: FC<SidebarNavSubItemProps> = ({
   children,
   href,
+  end,
```

Comment:
Maybe we could have the `SidebarNavSubItemProps` extend `NavLink` props instead of manually adding each prop. What do you think?

Reply:
I think for now I'd rather just say the few props that we need explicitly. `NavLink` has a lot of props, and some of them like `to` and `className` could get messy to forward.